From b3fee808d14a918072f5136209d71ae29e33591a Mon Sep 17 00:00:00 2001
From: Robby Cai <robby.cai@nxp.com>
Date: Wed, 18 May 2016 18:37:38 +0800
Subject: [PATCH 38/85] MLK-12811-4 ARM: dts: fix EPDC register region

privious size is overlapped with iomux_snvs. decrease it to fix it.

Signed-off-by: Robby Cai <robby.cai@nxp.com>
[zou:Original patch taken from
git://git.freescale.com/imx/linux-imx.git imx_4.1.15_2.0.0_ga]
Signed-off-by: zou cao <cao.zou@windriver.com>
---
 arch/arm/boot/dts/imx6ull.dtsi |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/boot/dts/imx6ull.dtsi b/arch/arm/boot/dts/imx6ull.dtsi
index eab5bbd..df69a78 100644
--- a/arch/arm/boot/dts/imx6ull.dtsi
+++ b/arch/arm/boot/dts/imx6ull.dtsi
@@ -1137,7 +1137,7 @@
 			epdc: epdc@0228c000 {
 				compatible = "fsl,imx7d-epdc";
 				interrupts = <GIC_SPI 112 IRQ_TYPE_LEVEL_HIGH>;
-				reg = <0x0228c000 0x10000>;
+				reg = <0x0228c000 0x4000>;
 				clocks = <&clks IMX6UL_CLK_EPDC_ACLK>,
 					 <&clks IMX6UL_CLK_EPDC_PIX>;
 				clock-names = "epdc_axi", "epdc_pix";
-- 
1.7.5.4

